spring-data-azure-documentdb/pom.xml

```diff
@@ -144,7 +144,7 @@
                         </goals>
                         <configuration>
                             <target>
-                                <echo message="skip test is ${skip.integration.tests}" />
+                                <echo message="skip test is ${skip.integration.tests}"/>
```

Comment:
could you pls help to delete this target? this is for debug purpose, forget to delete them by myself.. thanks